backend/tests/test_service_address.py

```diff
@@ -319,50 +305,14 @@ def test_many_addresses(self, m):
     @util.mock()
     def test_bad_scope(self, m):
         with self.assertRaisesRegex(exceptions.HTTPException,
-                                    'invalid address scope'):
-            address.get_one_address(
-                lora.Connector(),
-                {
-                    'objekttype': 'kaflaflibob',
-                    'uuid': '00000000-0000-0000-0000-000000000000',
-                },
-            )
-
-    @util.mock()
-    @freezegun.freeze_time('2010-01-01', tz_offset=1)
-    def test_bad_phone(self, m):
```

Comment:
I'm not really sure what this is actually testing.

Reply:
> I'm not really sure what this is actually testing.

The test consists of reading a value from an address that disagrees with the class; e.g. what happens when an address of type `EMAIL` has a phone number stored? Yeah, it's rather esoteric, but a useful thing to test.

Reply:
It's esoteric, yes, and I'm unsure of when it will actually be relevant. The addresses are created based on the scopes, so I can't see how the objects can suddenly become inconsistent.